repo-agent/examples/k8s-configdir.yaml

```diff
@@ -0,0 +1,716 @@
+apiVersion: configdir.gke.io/v1alpha1
+kind: ConfigDir
+metadata:
+  name: k8s-gemini-config
+spec:
+  files:
+  - path: .gemini/commands/document/package.toml
+    source:
```

Comment:
Nit: Starting to think we should consider having an optional description field for these to explain what the file is. Also seems odd the Kind is ConfigDir but the instance actually seems to be a file.